src/hooks/useCds/index.js

```diff
@@ -5,6 +5,9 @@ import { cdsResources } from 'services/fhir';
 import { valueSetJson } from 'services/valuesets';
 import { translateResponse, translateToggleChange } from './translate';
 import { stridesData } from './strides';
+import { logMsg } from 'util/logger';
+
+const LOGGER_ENABLED = process.env?.REACT_APP_LOGGER_ENABLED || true;
```

Comment:
Note: we should add new env variables to our wiki page: https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/wiki/Environment-Setting-for-Dashboard.

Reply:
Updated.